src/moonlink_service/src/rest_api.rs

```diff
@@ -220,7 +220,19 @@ async fn create_table(
         iceberg_config: None,
     };
     // Serialization not expect to fail.
-    let serialized_table_config = serde_json::to_string(&table_config).unwrap();
+    let serialized_table_config = match serde_json::to_string(&table_config) {
+        Ok(cfg) => cfg,
+        Err(e) => {
+            error!("serialize table_config failed: {}", e);
```

Comment:
For error handling, logging / propagation / panic / swallow are the general ways to deal with.
Usually one of them is enough, and here we use propagation;
Logging is fine is we provide extra context, but doesn't seem to necessary in this case.